plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -45,6 +45,9 @@
 
     private int insertBatchSize = 1000;
 
+    // Do not create temporary table during insert which ensures that all the rows will be inserted atomically.
```

Comment:
Misleading comment - enabling this makes the process non-atomic.

```suggestion
    // Do not create temporary table during insert. This means that the write operation can fail and leave the table in an inconsistent state.
```

Maybe copy this to the `ConfigDescription` too?

Reply:
I had the same feeling.

Reply:
Updated.